src/Monad/IO/functions.php

```diff
@@ -157,15 +157,15 @@ function getArgs()
     return M\IO::of(function () {
         if (!ini_get('register_argc_argv')) {
             throw userError(sprintf(
-                'argv is not available, because ini option "register_argc_argv" is disabled'
+                'argv is not available, because Widmogrod\ini option "register_argc_argv" is disabled'
```

Comment:
```diff
-'argv is not available, because Widmogrod\ini option "register_argc_argv" is disabled
+'argv is not available, because ini option "register_argc_argv" is disabled
```